onnxruntime/python/tools/transformers/models/phi2/README.md

```diff
@@ -0,0 +1,113 @@
+# Phi2 Optimizations
+## Prerequisites
+```bash
+pip install -r requirements.txt
+```
+_From source:_
+```bash
+pip install onnxruntime-gpu==1.17.0 # or onnxruntime==1.17.0 if using cpu

```

Comment:
This will install ORT for cuda 11. But requirements.txt install torch for CUDA 12.

Reply:
fixed